frontend/apps/reader/modules/readermenu.lua

```diff
@@ -53,7 +53,6 @@ function ReaderMenu:init()
     self.registered_widgets = {}
 
     if Device:hasKeys() then
-        self.key_events = { Close = { { "Back" }, doc = "close menu" }, }
```

Comment:
How will you close the menu?

Reply:
The back key is directly handled by touchemenu.